acslib/base/connection.py

```diff
@@ -137,16 +137,15 @@ def request(
         Parameters:
             requests_method: A method from the requests module. requests.get, requests.post, etc
             request_data: Data used as kwargs for the requests_method
-            timeout: Maximum time to wait for a server response, in seconds
 
         Returns: An object with status_code, json, and headers attributes
         """
 
         try:
-            # remove request_data_map properties with None values
             request_data_map = dict(request_data)
```

Comment:
Since this is a pydantic model calling request_data.model_dump() is the cannonical way to render the model.